python/perf-kernels/rmsnorm.py

```diff
@@ -87,21 +99,15 @@ def rmsnorm(x, epsilon=1e-6):
 
 
 def run_rmsnorm(M, N):
+    print(f"Running RMSNorm for shape ({M}, {N})")
     torch.manual_seed(0)
     x = torch.randn(M, N, device='cuda')
     y_triton = rmsnorm(x)
 
     return y_triton
 
 
-@pytest.mark.parametrize('M, N', [
-    (1, 4),
-    (2, 10),
-    (8192, 4096),
-    (4096, 8192),
-    (1, 8192),
-    (873, 1245),
-])
+@pytest.mark.parametrize('M, N', [(1, 4), (2, 10), (8192, 4096), (4096, 8192), (1, 8192), (873, 1245), (1, 98304)])
```

Comment:
Was this formatting change intentional? Or it was done my `yapf`? I really think the previous formatting is much more readable. You can do something like this to disable the auto formatter:
```python
# yapf: disable
@pytest.mark.parametrize('M, N', [
    (1, 4),
    (2, 10),
    (8192, 4096),
    (4096, 8192),
    (1, 8192),
    (873, 1245),
])
# yapf: enable
```

Reply:
This was done by yapf. Thanks for the tip!

Reply:
Fixed